ppocr/utils/save_load.py

```diff
@@ -109,9 +109,14 @@ def init_model(config, program, exe):
     """
     checkpoints = config['Global'].get('checkpoints')
     if checkpoints:
-        path = checkpoints
-        fluid.load(program, path, exe)
-        logger.info("Finish initing model from {}".format(path))
+        if os.path.exists(checkpoints + '.pdparams'):
+            path = checkpoints
+            fluid.load(program, path, exe)
+            logger.info("Finish initing model from {}".format(path))
+        else:
+            raise ValueError(
+                "Model checkpoints  {} does not exists,"
```

Comment:
`checkpoints  {}` -> `checkpoints {}` (delete one of the two blanks)
`prefix .` -> `prefix.`

Reply:
Done, this pr is in [https://github.com/PaddlePaddle/PaddleOCR/pull/467](url)